### PR TITLE
add $:/config/AutoFocus for conditional focusing of ...

### PR DESCRIPTION
**Pull request by BurningTreeC**
title, tags, editor, type or field inputs

### CODE REVIEW

**Comment by BurningTreeC** [comment]
@Jermolene, could this be considered for merging?

File: core/language/en-GB/ControlPanel.multids [hunk 1]
```diff
@@ -5,6 +5,7 @@ Advanced/Hint: Internal information about this TiddlyWiki
 Appearance/Caption: Appearance
 Appearance/Hint: Ways to customise the appearance of your TiddlyWiki.
 Basics/AnimDuration/Prompt: Animation duration:
+Basics/AutoFocus/Prompt: Default AutoFocus field:
```
Comment (Jermolene) [inline core/language/en-GB/ControlPanel.multids]: Perhaps "AutoFocus" shouldn't be camel case?

**Comment by BurningTreeC** [comment]
Hi @Jermolene, you've been a bit too fast with merging, I wanted to address your comment about `AutoFocus` being named `Autofocus`

**Comment by Jermolene** [comment]
Hi @BurningTreeC whoops! Apologies, if that's the only thing I can just do it as a direct commit?

**Comment by BurningTreeC** [comment]
@Jermolene , yes I'll create a separate PR

**Comment by Jermolene** [comment]
Thanks, and apologies

**Pull request merged**